listenbrainz/spark/request_manage.py

```diff
@@ -156,16 +156,18 @@ def request_model(rank, itr, lmbda, alpha):
 @click.option("--days", type=int, default=7, help="Request recommendations to be generated on history of given number of days")
 @click.option("--top", type=int, default=20, help="Calculate given number of top artist.")
 @click.option("--similar", type=int, default=20, help="Calculate given number of similar artist.")
+@click.option("--html", is_flag=False, help='Enable/disable HTML file generation')
```

Comment:
>is_flag=False

Should this be True? This is a flag, right?

Reply:
beat me to it. :)

Reply:
I wanted a flag whose default value should be `False`.

Any other way?

Reply:
Maybe click.bool

Reply:
is_flag=True, default=False

should do it. See https://click.palletsprojects.com/en/7.x/options/

Reply:
Thanks🌹